src/components/categoryFilter/CategoryFilter.tsx

```diff
@@ -35,7 +35,7 @@ export const CategoryFilter = <T,>({
 }: CategoryFilterProps & {
   entries: T[]
   render: (renderProps: {
-    entries: (predicate: (entry: T) => string) => T[]
+    entries: (predicate: (entry: T) => string | string[]) => T[]
```

Comment:
We'll require a new story (per `variant`) to highlight the usage if this is changing.

Reply:
I think this is covered with the new story in `CategoryFilter.stories` -> "checkbox with multi-category entries"?